Dockerfile

```diff
@@ -78,8 +78,7 @@ RUN ${CONDA_EXECUTABLE} env create -f ./danesfield/deployment/conda/conda_env.ym
 # Install core3d-tf_ops package from kitware-danesfield / defaults
 RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
   conda activate core3d && \
-  pip install pip==20.0.1 && \
-  conda install -c kitware-danesfield -c kitware-danesfield-df -y core3d-tf_ops && \
+  conda install -c kitware-danesfield -c kitware-danesfield-df -c conda-forge -c defaults -c pytorch -y core3d-tf_ops && \
```

Comment:
Wondering why the additional channels were needed here, also if there's way to install `core3d-tf_ops` here without installing it's dependencies (because presumably we've already installed them), that might remove one potential source of errors.